### PR TITLE
Session8

### PR DESCRIPTION
**Pull request by kumarmohit19**
session8 branch created with test scripts in it.

### CODE REVIEW

File: routes/category.routes.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const categoryController = require("../controllers/category.controller")
 module.exports = function(app){
     
     //Route for the POST request to create the category
-    app.post("/ecomm/api/v1/categories",[requestValidator.validateCategoryRequest,authJwt.verifyToken,authJwt.isAdmin ], categoryController.create);
```
Comment (Vishwa07dev) [inline routes/category.routes.js]: Why have you removed these validators ?

File: routes/product.routes.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
  module.exports = function(app){
      
      //Route for the POST request to create the product
-     app.post("/ecomm/api/v1/products",[requestValidator.validateProductRequest, authJwt.verifyToken,authJwt.isAdmin], productController.create);
```
Comment (Vishwa07dev) [inline routes/product.routes.js]: Why have you removed this validator ?

Comment (Vishwa07dev) [reply routes/product.routes.js]: same question at all the other places

File: tests/controllers/auth.controller.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        await AuthController.signup(req, res);
+        
+        expect(spy2).toHaveBeenCalled();
+        // expect(res.status).toHaveBeenCalledWith(201);
+        // expect(res.send).toHaveBeenCalledWith({ message: "User registered successfully!" });    
+    });
+});
+
+describe('Auth controller signup method', () => {
```
Comment (Vishwa07dev) [inline tests/controllers/auth.controller.test.js]: You mean sign in method ?

**Review by Vishwa07dev — COMMENTED** [review]
Can we avoid using sequelize-cli

**Comment by kumarmohit19** [comment]
I did not had any reference to look for implementing the test environment
for this project and neither enough time to complete it, one backend
project was there for reference but it didn't used ORM tool and there also
all validators were removed.

If you want the test can be refined but it will take some time and some
help as well.

Regards,
Mohit

**Comment by kumarmohit19** [comment]
All the above mentioned comments are resolved, and all almost 100% test coverage is implemented, kindly have a look

**Pull request merged**